package.json

```diff
@@ -18,7 +18,8 @@
     "watch:html": "onchange \"src/templates/*.html\" -- npm run build:html",
     "watch:css": "onchange \"src/scss/*.scss\" -- npm run build:css",
     "watch:js": "onchange \"src/js/*.js\" -- npm run build:js",
-    "watch:all": "concurrently --kill-others \"npm run serve\" \"npm run watch:css\" \"npm run watch:js\" \"npm run watch:html\""
+    "watch:all": "concurrently --kill-others \"npm run serve\" \"npm run watch:css\" \"npm run watch:js\" \"npm run watch:html\"",
+    "start": "npm run serve"
```

Comment:
I think the `npm start` command should trigger the `watch:all` script.. dunno, maybe I'm used to that. But it feels natural hehe